docs/samples/local/local_quickstart_nyc_taxi_demo.ipynb

```diff
@@ -0,0 +1,1277 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Feathr Local Spark Quickstart - NYC Taxi Demo\n",
+    "This notebook demonstrates how to use Feathr Local to train a model on a local Spark cluster. We will use the NYC Taxi dataset to predict the tip amount for a taxi ride. The dataset is available on [Kaggle](https://www.kaggle.com/c/new-york-city-taxi-fare-prediction/data)."
```

Comment:
This notebook demonstrates how to use Feathr Local to train a model on a local Spark cluster
===============
The first local can be removed